xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -778,20 +778,27 @@ workflow:
         </mediaobject>
       </figure>
 
-      <para>Moreover, if your package builds several multibuild flavours, the status will
+      <para>Moreover, if your package builds several multibuild flavors, the status will
```

Comment:
Any reason to change from `flavours` to `flavor`? It seems that `flavor` is used is the preferred spelling in the United States, while `flavour` is used the rest of the English-speaking world.

Reply:
In the SCM/CI integration, we used both. See below the line 783 with the link to https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.multibuild.html. On this page, we always use `flavor`, so this explains why I decided to go with `flavor`.

Reply:
I see. Thanks!